src/GeometryInstancing/Camera.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.Numerics;
+using System.Runtime.InteropServices;
+using Veldrid.Sdl2;
+
+namespace Veldrid.NeoDemo
+{
+    public class Camera : IUpdateable
```

Comment:
There's actually already a `Camera` class in the SampleBase project. Can we re-use that?

Reply:
Yes I can change that.